### ISSUE TITLE
[Feature Request] Video looping in the Media player

### ISSUE THREAD

**Issue by bodqhrohro** [telegramdesktop/tdesktop#17401]
### Is your feature request related to a problem?

Telegram for Android loops the videos by default.

TelegramDesktop doesn't seem to be able to loop them at all.

### Describe the solution you'd like

Loop the videos in the media viewer (either always or by an option).

### Describe alternatives you've considered

Well, open a video in an external player that supports looping, or hit the play button manually every time? ;)

### Additional context

C'mon, am I really first with this FR?

**Comment by GregorKopka** [comment]
Best would be to have it as an option in all ecosystems.
Preferably together with an option to 'not autoplay preview of videos'.

**Comment by r0ei** [comment]
Interesting how it isn't implemented yet

**Comment by Vitor12xx** [comment]
> Interesting how it wasn't implemented yet

To this date it isn't.... Unbelievable.

**Comment by enomado** [comment]
Need this

**Comment by anildalar** [comment]
Yes i need

**Comment by dafraer** [comment]
Still not implemented somehow

**Comment by theLucasAntunes** [comment]
Would love this.

**Comment by MakotoU7** [comment]
Must have functionality!

**Comment by TodoshiTakato** [comment]
3 years have passed, still not implemented in the Desktop version...

**Comment by Okloks** [comment]
need this

**Comment by qodroi** [comment]
It's now out for Telegram on Android; Until now Telegram would only loop short videos, now you can select the option to loop videos that are long. AFAIK Telegram on Desktop uses an external video player, so this feature cannot be applied to telegram itself but to your favorite video player.

**Comment by bodqhrohro** [comment]
> AFAIK Telegram on Desktop uses an external video player, so this feature cannot be applied to telegram itself but to your favorite video player.

No, it launches an external player only for video formats it doesn't support, otherwise, the internal Media Viewer is used.

**Comment by TodoshiTakato** [comment]
Guys, it's been 5 years already. Hope is gone.  
Time to infiltrate the Telegram dev team and add this feature ourselves 😂

**Comment by DenisKaz** [comment]
> Time to infiltrate the Telegram dev team and add this feature ourselves 😂
seems to be the best option =)

**Comment by enomado** [comment]
even if we do - i think i wont be merged b/c slop etc. but i'll try

**Comment by enomado** [comment]
Conflicts 👆

**Comment by DenisKaz** [comment]
> Conflicts 👆

sad(  seems to have lack of information and no way to solve these conflicts.
